### PR TITLE
Writing readme sections for testing

### PR DESCRIPTION
**Pull request by Eysteinn-Orn**
# Writing the sections in the Readme regarding testing.
Just added some text regarding the testing process, installing dependencies, running tests locally, writing tests, CI, Libraries used, when to test your code and testing guidelines.

### CODE REVIEW

**Review by lsig — APPROVED** [review]
Wow, great job on the documentation! 👍

File: README.md [hunk 1]
```diff
@@ -3,3 +3,109 @@
 
 # Finetune QA Powerset
 
+## Testing
+
+This project uses PyTest for unit testing, and Poetry for managing dependencies. Tests help ensure the integrity and reliability of the code, and they also serve as examples of how to use the various functions and classes. As you contribute, we strongly encourage you to add tests for your code.
```
Comment (njallskarp) [inline README.md:7]: nice

**Review by njallskarp — APPROVED** [review]
great

**Comment by njallskarp** [comment]
Approved. Just make one update first. Run the linter and formater (pylint and black) with the fix option to fix some of the 24 issues pointed out by codacy. Once that's good you can merge it

**Comment by Eysteinn-Orn** [comment]
So, the formatter we are using doesn't format markdown files, what I ended up doing was installing remark using node and running remark . --output using node and that formatted the markdown file as per Codeacy recommendations

**Pull request merged**